src/pages/manualLoggingSteps/manualLoggingSteps.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import {
+    IonButton,
+    IonCol,
+    IonContent,
+    IonHeader,
+    IonInput,
+    IonItem,
+    IonLabel,
+    IonNote,
+    IonRouterLink,
+    IonTitle,
+    IonToolbar,
+    IonGrid,
+    IonRow,
+} from '@ionic/react';
+//import ExploreContainer from '../../components/ExploreContainer';
+import './manualLoggingSteps.css';
+
+const ManualSteps: React.FC = () => {
+    const [username, setUsername] = useState();
+    const [password, setPassword] = useState();
+    const [isTouched, setIsTouched] = useState(false);
+    const [isValid, setIsValid] = useState<boolean>();
+    var stepsToLog: number;
+    var date: Date;
+    // var pastRecordDates: Array: Date;
+    var pastRecordDates: Array<Date>;
+    var pastRecordSteps: Array<number>;
+    interface Record {
+        time: Date;
+        stepsWaled: number;
+    }
+    var pastRecords: Array<Record>;
+
+    function writeToDatabase() {
+        if (stepsToLog > 0) {
+            console.log(stepsToLog);
```

Comment:
Can we put a TODO comment in here just so that we don't forget that this still needs to be implemented when the database is actually ready?

Reply:
Added TODO comment